### ISSUE TITLE
Dropdowns, ViewEncapsulation.Native

### ISSUE THREAD

**Issue by KingsDevelopment** [ng-bootstrap/ng-bootstrap#1039]
### Bug description:
Dropdowns stop working when using native shadow dom.

### Link to minimally-working plunker that reproduces the issue:
http://plnkr.co/edit/mbOsVfnOjeHtbxfpuGaR?p=preview

### Version of Angular, ng-bootstrap, and Bootstrap:
Angular: 2.1.0

ng-bootstrap: 1.0.0-alpha.10

Bootstrap: 4.0.0.aplha.5

**Comment by pkozlowski-opensource** [comment]
It is not specific to dropdowns - in fact using native view encapsulation prevents _any_ Bootstrap CSS from being visible so even basic things like alerts won't work: http://plnkr.co/edit/r1AT9DZFONGeU8KpPXTR?p=preview

I would argue that using `ViewEncapsulation.Native` with CSS frameworks that assume to be loaded and available globally (as Bootstrap) doesn't make much sense - you would have to load Bootstrap's CSS into each and every component which could have considerable perf impact.

In short: I don't think that Bootstrap's way of approaching things is compatible with `ViewEncapsulation.Native` and as such I'm going to close this issue as "won't fix".

**Comment by iodel** [comment]
What if we want to include bootstrap classes into an Angular Element which has to use `ViewEncapsulation.Native`  ? Of course, it doesn't make sense to include bootstrap in every component. It's weird because some styles like `font-family`  targeting HTML elements like `h1` etc are applied by _reboot.scss . However, the actual BS classes like .card, .jumbotron, .container etc do not work inside the Angular Element.
